Makefile

```
@@ -85,6 +85,55 @@ GO_CMD_PATHS := $(filter-out library, $(wildcard cmd/*))
 GO_CMD_NAMES := $(notdir $(GO_CMD_PATHS))
 GO_CMD_BUILDS := $(addprefix build/bin/, $(GO_CMD_NAMES))
 
+
+
+# Our custom config is located in nix/nix.conf
+export NIX_USER_CONF_FILES = $(PWD)/nix/nix.conf
+# Location of symlinks to derivations that should not be garbage collected
+export _NIX_GCROOTS = /nix/var/nix/gcroots/per-user/$(USER)/status-go
+
+
+#----------------
+# Nix targets
+#----------------
+
+
+SHELL := ./nix/scripts/shell.sh
+
+shell: ##@prepare Enter into a pre-configured shell
```

We're missing a default target. And you've made changes to `nix/scripts/shell.sh` that I don't understand.
```suggestion
shell: export TARGET ?= default
shell: ##@prepare Enter into a pre-configured shell
```